### PR TITLE
Fix/3790 update wording for join notification

### PR DESCRIPTION
**Pull request by gbonaspetti**
#3790 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/i18next.scanner/pt/translation.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "{{author}} added you to {{space}}": "{{author}} adicionou-o à {{space}}",
+  "{{author}} joined space {{space}}": "{{author}} juntou-se ao espaço {{space}}",
```
Comment (Skylsmoi) [inline frontend/i18next.scanner/pt/translation.json:3]: This Portuguese translation doesn't match the specified one (which is "X juntou o espaço Y" in the issue)

But I'm not qualified to judge which is the best

File: frontend/src/container/NotificationWall.jsx [hunk 2]
```diff
@@ -396,7 +406,11 @@ export class NotificationWall extends React.Component {
                 >
                   <span className='notification__list__item__icon'>{icon}</span>
                   <div className='notification__list__item__text'>
-                    <Avatar size={AVATAR_SIZE.MINI} publicName={notification.author} style={{ marginRight: '5px' }} />
+                    <Avatar
+                      size={AVATAR_SIZE.MINI}
+                      publicName={notification.author ? notification.author.publicName : 'unkown'}
```
Comment (Skylsmoi) [inline frontend/src/container/NotificationWall.jsx:411]: shouldn't unknown be translated ?
If "unknown" is handled specifically, it should be a constant

**Review by Skylsmoi — CHANGES_REQUESTED** [review]

File: frontend/i18next.scanner/pt/translation.json (hunk 1 above)
Comment (gbonaspetti) [reply frontend/i18next.scanner/pt/translation.json:3]: Issue updated.

File: frontend/src/container/NotificationWall.jsx (hunk 2 above)
Comment (gbonaspetti) [reply frontend/src/container/NotificationWall.jsx]: Updated

**Review by Skylsmoi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Test on branch
Screenshot_2021-01-14 (1) Painel · issue 3790 · Tracim
Screenshot_2021-01-14 (1) Tableau de bord · issue 3790 · Tracim
Screenshot_2021-01-14 (1) Dashboard · issue 3790 · Tracim

**Pull request merged**